### ISSUE TITLE
.meteor/release file

### ISSUE THREAD

**Issue by aspiringpro** [awatson1978/payment-subscription#1]
Problem! This project does not have a .meteor/release file. The file should
either contain the release of Meteor that you want to use, or the word 'none'
if you will only use the project with unreleased checkouts of Meteor. Please
edit the .meteor/release file in the project and change it to a valid Meteor
release or 'none'.

**Comment by awatson1978** [comment]
Ah, yes.  It's still on 0.6.5, can you believe it?  Trying to find the time to upgrade all the projects and demo apps.  Upgrading this is on my list!

**Comment by awatson1978** [comment]
This had been on my upgrade list for a couple of months now, and there are a couple of recent requests about it, so I went and updated it to Meteor 1.0.2.1.  However, the underlying stripe package that it relies on has gone through a bit of change over the past six months, so I moved it to the `mrgalaxy:stripe` package, and my initial charge attempt was declined.  So I suspect there is still some integration testing left to be done.  Feel free to submit more issues as you run across them, and I'll try to help clear things up.  In the meantime, project now has a .meteor/release file.
